src/components/calendar/calendar.material.scss

```diff
@@ -0,0 +1,228 @@
+@use '../../styles/utilities' as *;
+@use 'sass:map';
+@use 'sass:math';
+
+$date-size-large: rem(32px);
+$date-size-medium: rem(28px);
+$date-size-small: rem(24px);
+
+:host {
+    display: flex;
+    flex-direction: column;
+    font-family: var(--igc-font-family);
+
+    > * {
+        font-family: var(--igc-font-family);
+    }
+}
+
+:host([size='large']) {
+    igc-days-view::part(days-row) {
+        min-height: $date-size-large;
+    }
+
+    igc-days-view::part(week-number-inner),
+    igc-days-view::part(date-inner) {
+        width: $date-size-large;
+        min-width: $date-size-large;
+        height: $date-size-large;
+    }
+
+    igc-days-view::part(label) {
+        min-width: $date-size-large;
+        height: $date-size-large;
+    }
+
+    igc-days-view::part(date-inner) {
+        border-radius: math.div($date-size-large, 2) + rem(2px);
+    }
+
+    igc-months-view {
+        grid-template-rows: repeat(4, minmax($date-size-large, 1fr));
```

Comment:
I guess we should replace these with something else since we are no longer using CSS grid.